### DISCUSSION TITLE
Feedback UX-UI

### DISCUSSION THREAD

**Discussion by OmarVenturaP** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#182] [Q&A]
Buenas tardes compañeros, quisiera que me apoyaran haciendo una retroalimentación de mis practicas, ya que las comence sin tener ningún conocimiento de UX/UI, MIRO, FIGMA, etc. 

La verdad si me genero conflicto un poco usar para el UX un programa, me fui por "la vieja confiable" que es el boceto a mano, lo hice así:
https://github.com/OmarVenturaP/Practica-Frontend-Sem1/blob/main/04%20-%20UX%20ABOGABOT.pdf

Ya con el UI, al ser algo mas visual y sobre como lo vería el usuario, pues me aventé a unos pequeños tutos de Figma y pude hacer esta plantilla de Abogabot:
https://www.figma.com/file/cfo2idycNrsv3vD3PdD9cH/Abogabot-App?node-id=0%3A1

Les comento que aunque ya envié el link de las practicas, aun lo sigo mejorando, para tenerlo completo, pero a la vez quisiera de su apoyo para darme un Feedback sobre mi proyecto, para que eso me ayude a seguir mejorando.

De antemano les agradezco su apoyo.

**Comment by OscarIslasH** [comment]
A mi opinion te quedaron muy bien, si quieres hacer el Wireframe en miro, puedes usar plantillas que hay ahi, ya cuando cabe lo descubri y ni cuenta jajaja 

Los iconos puedes sacarlos de: https://tablericons.com/
Y tambien de : https://heroicons.com/

![Captura de pantalla 2022-03-01 152651](https://user-images.githubusercontent.com/99118114/156252155-de752372-c07e-4cd7-b82e-ef58d37f5a2a.png)


![Captura de pantalla 2022-03-01 152734](https://user-images.githubusercontent.com/99118114/156251862-f56ee37b-24c5-472a-8d81-de02636b67c5.png)


![Ade pantalla 2022-03-01 152651](https://user-images.githubusercontent.com/99118114/156252318-1d308ca2-74df-4e1d-81de-0423c275a0fe.png)


![Captura de pantalla 2022-03-01 152915](https://user-images.githubusercontent.com/99118114/156252011-d823ced7-d4c4-45d3-b649-db93d50c6719.png)

**Reply by OmarVenturaP** [reply]
Que buen dato, voy a tratar de hacerlo allí, créeme que no había visto esa opción desde miro, pero se ve bastante útil. Gracias por el apoyo.

**Reply by OscarIslasH** [reply]
No hay de que, en la parte de iconfinder puedes encontrar iconos, pero te recomiendo mas en la pagina de tablericons por que ahi te deja elegir los colores de los simbolos y en Wireframe Library puedes encontrar la plantilla de telefono, de una pagina web, de botones o de iconos

**Comment by JuanSalvador73** [comment]
El wireframe a mano - vieja escuela - te quedó muy bien. Básicamente es para que como desarrollador tenga uno muy claro como es el proceso de la App en su lógica de trabajo, si están bien delimitados los pasos estás del otro lado.

El UI es ese mismo proceso pero ya con el diseño gráfico tal y como va a quedar. Generalmente este es el que se suele compartir con el equipo de trabajo o incluso presentaciones con el cliente.

Es muy común que en tu equipo - si es el caso - intervenga en la creación del UX porque así cada miembro puede aportar ideas que ayuden a desarrollar un proyecto muy claro y conciso desde el principio. 

Los diagramas te quedaron muy bien. Saludos. :D
